src/resources/views/crud/fields/relationship/modal.blade.php

```diff
@@ -0,0 +1,57 @@
+@php
+
+    // in parent form_content we added those fields to the session,
+    // here we check if there are any, and mark them as loaded
+    // so we don't get assets that are already on page from parent crud
+    if (session()->has('current_crud_loaded_fields')) {
+        $loadedFields = session('current_crud_loaded_fields');
+        session()->forget('current_crud_loaded_fields');
+    }
+
+    $loadedFields = $loadedFields ?? [];
+
+    //mark parent crud fields as loaded.
+    foreach($loadedFields as $loadedField) {
+        $crud->markFieldTypeAsLoaded($loadedField);
+    }
+
+@endphp
+<div class="modal fade" id="{{$entity}}-inline-create-dialog" tabindex="-1" role="dialog" aria-labelledby="{{$entity}}-inline-create-dialog-label" aria-hidden="true">
+    <div class="modal-dialog" role="document">
```

Comment:
## Relationship Issue 10

A normal modal works well for small entities, but for more complicated forms in modals (including tabs, for ex), the default modal is too small:
![Screenshot 2020-03-12 at 15 23 41](https://user-images.githubusercontent.com/1032474/76525963-7d0a4e00-6475-11ea-942c-13e074c51223.png)

We could give developers the option to change the size of the modal dialog, by adding the ```modal-lg``` or ```modal-xl``` classes next to ```modal-dialog```. Since this modal is only used by the inlineCreate operation, maybe a good syntax for this would be:
```diff
            $this->crud->addField([    // Relationship
                'label'     => 'Relationship (n-n with InlineCreate; Fetch using AJAX) <span class="badge badge-warning">New in 4.1</span>',
                'type'      => 'relationship',
                'name'      => 'products',
                'entity'    => 'products',
                'attribute' => 'name',
                // 'tab'       => 'Selects',
                'ajax' => true,
                'inline_create' => [
                      'entity' => 'product',
+                     'modalClass' => 'modal-dialog modal-lg',
                ],
                'data_source' => url($this->crud->getRoute().'/fetch/product'),
            ]);
```

What do you think @pxpm ? What would be a good way to have this configurable?

Reply:
I think this is a good way of dealing with dialog resizing. Just implemented it and it works great.

Reply:
Confirmed! Excellent. Leaving open so we don't forget to include it in the docs.

You're right, ```modal_class``` makes more sense than ```modalClass``` - everything except ```wrapperAttributes``` is snake_case... I hate that 😠 But then again, https://github.com/Laravel-Backpack/CRUD/issues/2474 would fix it...

Reply:
Though, if you were to choose ONE programming language where the developers are _already used to_ having some methods ```camelCase``` and others ```snake_case```  - it would be PHP 🤣

Reply:
Done - added docs for this https://github.com/Laravel-Backpack/Docs/commit/65d899069600473e3ce7df601beab67c110bcb2d